Same error with all LAS files.  DeallocateProjection

Hello! 
I have the same error with all my las files.
Do you have any working las file?
Best regards!!

Preparing list of files to process...
Processing file 1/1
> reading data from las file... stadium_all.las
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x4d135d]

goroutine 1 [running]:
github.com/xeonx/proj4.(*Proj).Close(0x0)
        C:/Users/user1/go/src/github.com/xeonx/proj4/proj.go:70 +0x2d
github.com/mfbonfigli/gocesiumtiler/converters.DeallocateProjection(...)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/converters/converters.go:133
github.com/mfbonfigli/gocesiumtiler/lasread.(*LasFile).readPointsOctElem(0xc00008ac80, 0xc000042370, 0x10e6, 0x5d21e0, 0xc000448060, 0x0, 0x0)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/lasread/tiler_las_reader.go:178 +0x45d
github.com/mfbonfigli/gocesiumtiler/lasread.(*LasFile).readForOctree(0xc00008ac80, 0xc000042370, 0x10e6, 0x5d21e0, 0xc000448060, 0x0, 0x0)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/lasread/tiler_las_reader.go:61 +0x21c
github.com/mfbonfigli/gocesiumtiler/lasread.NewLasFileForTiler(0xc00000e367, 0x21, 0xc000042370, 0x10e6, 0x5d21e0, 0xc000448060, 0xc000006018, 0xc00007dd68, 0x2)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/lasread/tiler_las_reader.go:26 +0x12f
main.readLas(0xc00000e367, 0x21, 0xc000042370, 0xc000018380, 0x5d21e0, 0xc000448060, 0x0, 0x0)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/tiler.go:125 +0x97
main.loadLasInOctree(...)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/tiler.go:114
main.RunTiler(0xc000018380, 0x0, 0x0)
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/tiler.go:84 +0x56e
main.main()
        C:/Users/user1/go/src/github.com/mfbonfigli/gocesiumtiler/main.go:99 +0x8f9

--- a/.gitignore
+++ b/.gitignore
@@ -14,4 +14,5 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 .idea/workspace.xml
+.idea/encodings.xml
 output/

--- a/converters/converters.go
+++ b/converters/converters.go
@@ -129,7 +129,7 @@ func initProjection(code int) (*proj.Proj, error) {
 // Releases a projection object from memory
 func DeallocateProjection(code int) {
 	val, ok := EpsgDatabase[code]
-	if ok {
+	if ok && val.Projection != nil {
 		val.Projection.Close()
 		EpsgDatabase[code].Projection = nil;
 	}

